src/components/Table/Table.jsx

```diff
@@ -40,8 +42,8 @@ const Table = ({ columnsConfig, data, totalAmount, onClickRow, onDataRefresh })
     }
   };
 
-  const handleChangeAmountEl = (event) => {
-    const amount = parseInt(event.target.value, 10);
+  const handleChangeAmountEl = (value) => {
+    const amount = parseInt(value, 10);
```

Comment:
You could pass it as a number from your component, In that case, you'll not need to parse it every time